client/src/ledger_app_clients/ethereum/eip712/InputData.py

```diff
@@ -264,32 +271,84 @@ def send_filtering_message_info(display_name: str, filters_count: int):
     to_sign += sig_ctx["caddr"]
     to_sign += sig_ctx["schema_hash"]
     to_sign.append(filters_count)
-    for char in display_name:
-        to_sign.append(ord(char))
+    to_sign += display_name.encode()
 
     sig = keychain.sign_data(keychain.Key.CAL, to_sign)
     with app_client.eip712_filtering_message_info(display_name, filters_count, sig):
         enable_autonext()
     disable_autonext()
 
 
-# ledgerjs doesn't actually sign anything, and instead uses already pre-computed signatures
-def send_filtering_show_field(display_name):
+def send_filtering_amount_join_token(token_idx: int):
     global sig_ctx
 
     path_str = ".".join(current_path)
 
     to_sign = bytearray()
-    to_sign.append(72)
+    to_sign.append(11)
+    to_sign += sig_ctx["chainid"]
+    to_sign += sig_ctx["caddr"]
+    to_sign += sig_ctx["schema_hash"]
```

Comment:
Those lines seems the same in other functions. Can it be centralize into a function?

Reply:
Done :+1: